README.md

```diff
@@ -6,7 +6,11 @@ A command line interface to interact with [Rainforest QA](https://www.rainforest
 
 This is the easiest way to integrate Rainforest with your deploy scripts or CI server. See [our documentation](https://help.rainforestqa.com/docs/rainforest-cli-for-continuous-integration) on the subject.
 
-The CLI uses the Rainforest API which is documented at https://app.rainforestqa.com/docs
+The CLI uses the Rainforest API which is documented at https://help.rainforestqa.com/reference.
```

Comment:
ReadMe redirects to the first page listed in the `Reference` section. That is currently the `Authentication` page, but it could be any page we wish. Before I added the `Authentication` page, it was the `POST /callback` docs, since all we had at that point was endpoint documentation and that was the first endpoint (since the namespaces are sorted alphabetically).

So it's just a matter of adding a new `Intro to the API` page.

Reply:
Ok - @magni- can you work with @Nathan-RF to do that? It doesn't have to be an epic poem or anything, just something that says what the API does and why we have it and why the reader should care?